### PR TITLE
[ODS-4614] Cloud ODS Build is broken for 5.1.0

### PR DESCRIPTION
**Pull request by tbmeador-dlp**
new build can be found here: [Cloud ODS/API: Build, Package](https://intedfitools1.msdf.org/buildConfiguration/OdsPlatform_OdsApi_Cloud_CloudOdsApiBuildPackage/108740?buildTab=artifacts)

### CODE REVIEW

File: Scripts/NuGet/EdFi.RestApi.Databases/prep-package-cloud.ps1 [hunk 1]
```diff
@@ -23,6 +23,8 @@ param(
     $outputDirectory
 )
 
+Write-Warning "'Scripts\NuGet\EdFi.RestApi.Databases\prep-package-cloud.ps1' is deprecated, and will be removed in the near future. Functionality is now included in 'logistics\scripts\activities\build\CloudOds\PrepareDatabasesForExport.ps1' instead."
```
Comment (gmcelhanon) [inline Scripts/NuGet/EdFi.RestApi.Databases/prep-package-cloud.ps1:26]: More style than anything, this warning uses back-slashes, while others use forward slashes. If we care for this deprecation warning, then you should change it, but I'm not sure we do.

File: logistics/scripts/activities/build/initialize-database/initialize-empty-database.ps1 [hunk 2]
```diff
@@ -7,8 +7,10 @@
 $ErrorActionPreference = 'Stop'
 $Error.Clear()
 
-& "$PSScriptRoot\..\..\..\..\..\logistics\scripts\modules\load-path-resolver.ps1"
-Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'Application\SolutionScripts\InitializeDevelopmentEnvironment.psm1')
+& "$PSScriptRoot/../../../../../logistics/scripts/modules/load-path-resolver.ps1"
+Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1')
+
+Write-Warning "logistics/scripts/activities/build/initialize-database/initialize-empty-database.ps1 is deprecated, and will be removed in the near future. Please use the 'Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1' instead."
```
Comment (gmcelhanon) [inline logistics/scripts/activities/build/initialize-database/initialize-empty-database.ps1:13]: More a question of style, this warning doesn't wrap the embedded path in the message inside single quotes like other messages.

File: logistics/scripts/activities/build/CloudOds/PrepareDatabasesForExport.ps1 [hunk 3]
```diff
@@ -15,35 +15,31 @@
...
 
+if (-not (Test-Path $sqlPackagePath)) { throw "Could not find sqlpackage.exe at $sqlPackagePath" }
+
 $ErrorActionPreference = 'Stop'
 
-$repositoryNames = @('Ed-Fi-Ods', 'Ed-Fi-ODS-Implementation', 'Ed-Fi-ODS-Tools\Application\EdFi.Ods.AdminApp.Web')
-& "$PSScriptRoot/../../../../../logistics/scripts/modules/load-path-resolver.ps1" $repositoryNames
-$implementationRepo = Get-Item "$PSScriptRoot/../../../../.." | Select-Object -Expand Name
-$env:toolsPath = $toolsPath = (Join-Path (Get-RepositoryRoot $implementationRepo) 'tools')
+$repositoryNames = @('Ed-Fi-Ods', 'Ed-Fi-ODS-Implementation', 'Ed-Fi-ODS-AdminApp/Application/EdFi.Ods.AdminApp.Web')
```
Comment (gmcelhanon) [inline logistics/scripts/activities/build/CloudOds/PrepareDatabasesForExport.ps1:39]: I've noticed a lot of switch back and forth between single quotes and double quotes. Is there any particular style defined as "standard"?

File: logistics/scripts/activities/build/initialize-database/initialize-test-database.ps1 [hunk 4]
```diff
@@ -10,6 +10,8 @@ $Error.Clear()
 & "$PSScriptRoot\..\..\..\..\..\logistics\scripts\modules\load-path-resolver.ps1"
 Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'Application\SolutionScripts\InitializeDevelopmentEnvironment.psm1')
 
+Write-Warning "logistics/scripts/activities/build/initialize-database/initialize-test-database.ps1 is deprecated, and will be removed in the near future. Please use the 'Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1' instead."
```
Comment (gmcelhanon) [inline logistics/scripts/activities/build/initialize-database/initialize-test-database.ps1:13]: More a question of style, this warning doesn't wrap the embedded path in the message inside single quotes like other messages.

**Review by gmcelhanon — APPROVED** [review]
I don't see anything that jumps out at me as needing to be changed, but I did add some style comments. @tbmeador-dlp - let me know if you plan to make any changes. Otherwise, seems merge-worthy.

File: logistics/scripts/activities/build/CloudOds/PrepareDatabasesForExport.ps1 (hunk 3 above)
Comment (tbmeador-dlp) [reply logistics/scripts/activities/build/CloudOds/PrepareDatabasesForExport.ps1:39]: double quotes allows for variable substitution. The C# equivalent would be wrapping every string with `$""` even when not using string interpolation. Powershell doesn't warn you when you are using double quotes and no variables, but I try to fix them where I can.

File: Scripts/NuGet/EdFi.RestApi.Databases/prep-package-cloud.ps1 (hunk 1 above)
Comment (tbmeador-dlp) [reply Scripts/NuGet/EdFi.RestApi.Databases/prep-package-cloud.ps1]: I've updated all these warning to make them more consistent.

**Pull request merged**